--- /datas/dev/moodle/moodle_gitworkspaces/moodle404/moodle2_version_org/backup/moodle2/restore_stepslib.php	2022-09-30 04:17:34.000000000 +0200
+++ /datas/dev/moodle/moodle_gitworkspaces/moodle404/moodle2_version/backup/moodle2/restore_stepslib.php	2022-10-04 16:31:22.908913291 +0200
@@ -5554,7 +5556,9 @@
                 // This is a reference to some external file such as dropbox.
                 // If we are restoring to the same site, keep the reference untouched and
                 // restore the alias as is.
-                if ($this->task->is_samesite()) {
+                // UNISTRA DEV.
+                if ($this->task->is_samesite() or $info->oldfile->repositorytype == 'pod') {
+                //End UNISTRA DEV
                     $fs->create_file_from_reference($info->newfile, $repository->id, $info->oldfile->reference);
                     $this->notify_success($info);
                     continue;
@@ -5616,7 +5620,9 @@

             // Both Server files and Legacy course files repositories have a single
             // instance at the system context to use. Let us try to find it.
-            if ($info->oldfile->repositorytype === 'local' or $info->oldfile->repositorytype === 'coursefiles') {
+            // UNISTRA DEV.
+            if ($info->oldfile->repositorytype === 'local' or $info->oldfile->repositorytype === 'coursefiles' or $info->oldfile->repositorytype == 'pod') {
+            // END UNISTRA DEV.
                 $sql = "SELECT ri.id
                           FROM {repository} r
                           JOIN {repository_instances} ri ON ri.typeid = r.id
